test/Microsoft.DotNet.Tools.Api/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text.Encodings.Web;
+using System.Text.Unicode;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Tools.Api
+{
+    public class Program
+    {
+        public static string HttpGet(string requestUri)
+        {
+            try
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(requestUri);
```

Comment:
Use `HttpClient` unless infeasible. `WebRequest` is old and busted :). Better to show best practices in our repos.